src/style.css

```diff
@@ -1 +1,115 @@
-/* empty source */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 400;
+  line-height: 14px;
+  font-size: 10px;
+  color: #616070;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+
+  width: 200px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3 ;
+}
+
+.container__image {
+  height: 134px;
+  margin: 0 auto 40px;
+  width: 160px;
+}
+
+.container__text {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin-bottom: 4px;
+  color: #060b35;
+}
+
+.container__product-code {
+  margin-bottom: 16px;
+}
+
+.container__stars {
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position-y: center;
+  background-position-x: center;
+
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  margin-bottom: 24px;
```

Comment:
that is weird - to apply a margin to an individual star instead of applying margin to a flex container of both stars and rating number.
Please fix.

<img width="1744" alt="Monosnap Product cards 2022-11-08 11-42-57" src="https://user-images.githubusercontent.com/39308717/200531383-e3efdd47-fe79-41b8-9d4a-db2738c34231.png">